README.md

```diff
@@ -8,11 +8,17 @@ Add todo task on Habitica.com when task are added and complete task on Habitica.
 ## What it does not do
 - Add task in Taskwarrior added through Habitica
 
+## Install taskw
+This updated version of the hooks require the taskw [https://github.com/ralphbean/taskw](https://github.com/ralphbean/taskw) python package. It can be installed with pip:
+      pip install taskw
 
-## Install
+Or using your system package manager. For example, on Ubuntu (and Ubuntu-ish) Linux:
+      sudo apt install python-taskw
+
+## Install these hooks
 Download and unzip files
 
-	wget https://github.com/fplourde/Taskwarrior-habitica-hooks/archive/master.zip
+	wget https://github.com/ksonney/Taskwarrior-habitica-hooks/archive/master.zip
```

Comment:
Do not link to a different project. Make the modification and I will merge the code.